appengine/flexible/tasks/README.md

```diff
@@ -0,0 +1,105 @@
+# Google Cloud Tasks App Engine Queue Samples
```

Comment:
Do any of the instances of "App Engine" in this document need to be "App Engine Standard" or "App Engine Flexible"?  If not, can we actually explicitly say that "works in both App Engine Standard and Flexible"?  I work at Google and Idon't know, so I can only imagine our customers might have the same question.

Reply:
It works for both standard and flex.